src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/services/PlanService.kt

```diff
@@ -173,9 +173,13 @@ class PlanService(
    * Returns the PlanVersion which has been signed, not the current PlanVersion.
```

Comment:
presumably this doc line is no longer correct

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/117/commits/3ce560bd407082f1b6f6b2151202aaad83a721dc